mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -445,7 +450,7 @@ def _parallelized_download_from_cloud(
 
             if failed_downloads:
                 new_cloud_creds = self._get_read_credential_infos(
-                    self.run_id, dst_run_relative_artifact_path
+                    self.run_id, [dst_run_relative_artifact_path]
```

Comment:
This was broken before. `_get_read_credential_infos` expects a list of file paths. by passing a single file path as a string, the path was broken into individual characters, resulting in incorrectly-retried downloads